conda/utils.py

```diff
@@ -352,12 +352,9 @@ def wrap_subprocess_call(
         dev_mode,
         debug_wrapper_scripts,
         arguments,
-        use_system_tmp_path=False):
+):
     arguments = massage_arguments(arguments)
-    if not use_system_tmp_path:
-        tmp_prefix = abspath(join(prefix, '.tmp'))
-    else:
-        tmp_prefix = None
+    tmp_prefix = ".tmp"
```

Comment:
I think some explanation is required here. I came to the same conclusion as https://github.com/conda/conda/issues/10690#issuecomment-1031429467 and adjusted the `prefix` that is passed into `Utf8NamedTemporaryFile` so that it does not contain an absolute path - and therefore does not specify a location.